gst/datarepo/gstdatareposrc.c

```diff
@@ -426,25 +386,191 @@ gst_data_repo_src_read_tensors (GstDataRepoSrc * src, GstBuffer ** buffer)
   return GST_FLOW_ERROR;
 }
 
+
+/**
+ * @brief Get image filename
+ */
+static gchar *
+gst_data_repo_src_get_image_filename (GstDataRepoSrc * src)
+{
+  gchar *filename = NULL;
+  GST_ERROR ("media_type: %d", src->media_type);
```

Comment:
How about changing `GST_ERROR` to `GST_INFO` or `GST_DEBUG` ?

Reply:
Thank you.